--- 1128-antes
+++ 1128-despues
@@ -233,3 +233,3 @@
 <li>White A et al. <em>IGF-1 LR3 does not promote growth in late-gestation growth-restricted fetal sheep.</em> Am J Physiol Endocrinol Metab, 2025. PMID 39679943.</li>
-<li>Dunaiski V et al. <em>Long [R3] insulin-like growth factor-I reduces growth, plasma growth hormone, IGF binding protein-3 and endogenous IGF-I concentrations in finisher pigs.</em> J Endocrinol, 1998. PMID 9488001.</li>
+<li>Dunaiski V et al. <em>Long [R3] insulin-like growth factor-I reduces growth, plasma growth hormone, IGF binding protein-3 and endogenous IGF-I concentrations in finisher pigs.</em> J Endocrinol, 1997. PMID 9488001.</li>
 <li>Conlon MA et al. <em>Long R3 insulin-like growth factor-I (IGF-I) infusion stimulates organ growth but reduces plasma IGF-I, IGF-II and IGF binding protein concentrations in the guinea pig.</em> J Endocrinol, 1995. PMID 7561636.</li>
@@ -239,3 +239,3 @@
 <li>Salvi R et al. <em>N-Linked Glycosylation in Chinese Hamster Ovary Cells Is Critical for Insulin-like Growth Factor 1 Signaling.</em> Int J Mol Sci, 2022. PMID 36499281.</li>
-<li>Stremming J et al. <em>Sheep recombinant IGF-1 promotes organ-specific growth in fetal sheep.</em> Front Physiol, 2026. PMID 36091374.</li>
+
 </ol>