--- 1699-antes
+++ 1699-despues
@@ -286,10 +286,10 @@
 <ol>
-<li>Vyunova TV et al. <em>Peptide-based Anxiolytics: The Molecular Aspects of Heptapeptide Selank Biological Activity.</em> Protein Pept Lett, 2019. PMID 30255741.</li>
+<li>Vyunova TV et al. <em>Peptide-based Anxiolytics: The Molecular Aspects of Heptapeptide Selank Biological Activity.</em> Protein Pept Lett, 2018. PMID 30255741.</li>
 <li>Zozulia AA et al. <em>Efficacy and possible mechanisms of action of a new peptide anxiolytic selank in the therapy of generalized anxiety disorders and neurasthenia.</em> Zh Nevrol Psikhiatr Im S S Korsakova, 2008. PMID 18454096.</li>
 <li>Medvedev VE et al. <em>A comparison of the anxiolytic effect and tolerability of selank and phenazepam in the treatment of anxiety disorders.</em> Zh Nevrol Psikhiatr Im S S Korsakova, 2014. PMID 25176261.</li>
-<li>Panikratova YR et al. <em>Functional Connectomic Approach to Studying Selank and Semax Effects.</em> Dokl Biol Sci, 2021. PMID 32342318.</li>
-<li>Kolik LG et al. <em>Selank, Peptide Analogue of Tuftsin, Protects Against Ethanol-Induced Memory Impairment by Regulating of BDNF Content in the Hippocampus and Frontal Cortex.</em> Bull Exp Biol Med, 2020. PMID 31625062.</li>
+<li>Panikratova YR et al. <em>Functional Connectomic Approach to Studying Selank and Semax Effects.</em> Dokl Biol Sci, 2020. PMID 32342318.</li>
+<li>Kolik LG et al. <em>Selank, Peptide Analogue of Tuftsin, Protects Against Ethanol-Induced Memory Impairment by Regulating of BDNF Content in the Hippocampus and Frontal Cortex.</em> Bull Exp Biol Med, 2019. PMID 31625062.</li>
 <li>Kasian A et al. <em>Peptide Selank Enhances the Effect of Diazepam in Reducing Anxiety in Unpredictable Chronic Mild Stress Conditions in Rats.</em> Behav Neurol, 2017. PMID 28280289.</li>
 <li>Konstantinopolsky MA et al. <em>Selank, a Peptide Analog of Tuftsin, Attenuates Aversive Signs of Morphine Withdrawal in Rats.</em> Bull Exp Biol Med, 2022. PMID 36322304.</li>
-<li>Kolik LG et al. <em>Efficacy of peptide anxiolytic selank during modeling of withdrawal syndrome in rats with stable alcoholic motivation.</em> Bull Exp Biol Med, 2015. PMID 24913576.</li>
+<li>Kolik LG et al. <em>Efficacy of peptide anxiolytic selank during modeling of withdrawal syndrome in rats with stable alcoholic motivation.</em> Bull Exp Biol Med, 2014. PMID 24913576.</li>
 <li>Kolomin T et al. <em>The temporary dynamics of inflammation-related genes expression under tuftsin analog Selank action.</em> Mol Immunol, 2014. PMID 24291245.</li>