--- 19-antes
+++ 19-despues
@@ -249,3 +249,3 @@
 <li>Bajaj HS et al. <em>Efficacy and safety of retatrutide in people with type 2 diabetes and inadequate glycaemic control.</em> Lancet, 2026. PMID 42250575.</li>
-<li>Giblin K et al. <em>Retatrutide for the treatment of obesity, obstructive sleep apnea and knee osteoarthritis: Rationale and design of the TRIUMPH programme.</em> Diabetes Obes Metab, 2025. PMID 41090431.</li>
+<li>Giblin K et al. <em>Retatrutide for the treatment of obesity, obstructive sleep apnea and knee osteoarthritis: Rationale and design of the TRIUMPH programme.</em> Diabetes Obes Metab, 2026. PMID 41090431.</li>
 <li>Katsi V et al. <em>Retatrutide — A Game Changer in Obesity Pharmacotherapy.</em> Biomolecules, 2025. PMID 40563436.</li>
@@ -253,3 +253,3 @@
 <li>Kaur M et al. <em>A review of an investigational drug retatrutide, a novel triple agonist agent for the treatment of obesity.</em> Eur J Clin Pharmacol, 2024. PMID 38367045.</li>
-<li>Marathe SJ et al. <em>Incretin triple agonist retatrutide (LY3437943) alleviates obesity-associated cancer progression.</em> npj Metab Health Dis, 2026. PMID 40094000.</li>
+<li>Marathe SJ et al. <em>Incretin triple agonist retatrutide (LY3437943) alleviates obesity-associated cancer progression.</em> npj Metab Health Dis, 2025. PMID 40094000.</li>
 </ol>