--- 2232-antes
+++ 2232-despues
@@ -263,11 +263,11 @@
 <li>Teichman SL et al. <em>Prolonged stimulation of growth hormone (GH) and insulin-like growth factor I secretion by CJC-1295, a long-acting analog of GH-releasing hormone, in healthy adults.</em> J Clin Endocrinol Metab, 2006. PMID 16352683.</li>
-<li>Ionescu M et al. <em>Pulsatile secretion of growth hormone (GH) persists during continuous stimulation by CJC-1295, a long-acting GH-releasing hormone analog.</em> J Clin Endocrinol Metab, 2007. PMID 17018654.</li>
+<li>Ionescu M et al. <em>Pulsatile secretion of growth hormone (GH) persists during continuous stimulation by CJC-1295, a long-acting GH-releasing hormone analog.</em> J Clin Endocrinol Metab, 2006. PMID 17018654.</li>
 <li>Alba M et al. <em>Once-daily administration of CJC-1295, a long-acting growth hormone-releasing hormone (GHRH) analog, normalizes growth in the GHRH knockout mouse.</em> Am J Physiol Endocrinol Metab, 2006. PMID 16822960.</li>
-<li>Andersen NB et al. <em>The growth hormone secretagogue ipamorelin counteracts glucocorticoid-induced decrease in bone formation of adult rats.</em> Growth Horm IGF Res, 2002. PMID 11735244.</li>
-<li>Jiménez-Reina L et al. <em>Influence of chronic treatment with the growth hormone secretagogue Ipamorelin, in young female rats: somatotroph response in vitro.</em> Histol Histopathol, 2003. PMID 12168778.</li>
+<li>Andersen NB et al. <em>The growth hormone secretagogue ipamorelin counteracts glucocorticoid-induced decrease in bone formation of adult rats.</em> Growth Horm IGF Res, 2001. PMID 11735244.</li>
+<li>Jiménez-Reina L et al. <em>Influence of chronic treatment with the growth hormone secretagogue Ipamorelin, in young female rats: somatotroph response in vitro.</em> Histol Histopathol, 2002. PMID 12168778.</li>
 <li>Lu Z et al. <em>The growth hormone secretagogue receptor 1a agonists, anamorelin and ipamorelin, inhibit cisplatin-induced weight loss.</em> Physiol Behav, 2024. PMID 39043357.</li>
-<li>Aagaard NK et al. <em>Growth hormone and growth hormone secretagogue effects on nitrogen balance and urea synthesis in steroid treated rats.</em> Growth Horm IGF Res, 2010. PMID 19231263.</li>
+<li>Aagaard NK et al. <em>Growth hormone and growth hormone secretagogue effects on nitrogen balance and urea synthesis in steroid treated rats.</em> Growth Horm IGF Res, 2009. PMID 19231263.</li>
 <li>Timms M et al. <em>An immuno polymerase chain reaction screen for the detection of CJC-1295 and other growth-hormone-releasing hormone analogs.</em> Drug Test Anal, 2019. PMID 30489688.</li>
-<li>Memdouh S et al. <em>Advances in the detection of growth hormone releasing hormone synthetic analogs.</em> Drug Test Anal, 2022. PMID 34665524.</li>
-<li>Henninge J et al. <em>Identification of CJC-1295, a growth-hormone-releasing peptide, in an unknown pharmaceutical preparation.</em> Drug Test Anal, 2011. PMID 21204297.</li>
+<li>Memdouh S et al. <em>Advances in the detection of growth hormone releasing hormone synthetic analogs.</em> Drug Test Anal, 2021. PMID 34665524.</li>
+<li>Henninge J et al. <em>Identification of CJC-1295, a growth-hormone-releasing peptide, in an unknown pharmaceutical preparation.</em> Drug Test Anal, 2010. PMID 21204297.</li>
 </ol>