--- 799-antes
+++ 799-despues
@@ -229,8 +229,8 @@
 <li>Le NT et al. <em>Benzyl Alcohol: Allergen of the Year 2026.</em> Dermatitis, 2026. PMID 41649135.</li>
-<li>Kubin ME et al. <em>Benzyl alcohol allergy mimicking corticosteroid allergy.</em> Contact Dermatitis, 2017. PMID 27264293.</li>
-<li>Johnson W et al. <em>Safety Assessment of Benzyl Alcohol, Benzoic Acid and its Salts, and Benzyl Benzoate.</em> Int J Toxicol, 2018. PMID 29243541.</li>
+<li>Kubin ME et al. <em>Benzyl alcohol allergy mimicking corticosteroid allergy.</em> Contact Dermatitis, 2016. PMID 27264293.</li>
+<li>Johnson W et al. <em>Safety Assessment of Benzyl Alcohol, Benzoic Acid and its Salts, and Benzyl Benzoate.</em> Int J Toxicol, 2017. PMID 29243541.</li>
 <li>Behme RJ et al. <em>Incompatibility of ifosfamide with benzyl-alcohol-preserved bacteriostatic water for injection.</em> Am J Hosp Pharm, 1988. PMID 3369469.</li>
 <li>Nahata MC et al. <em>Stability of lorazepam diluted in bacteriostatic water for injection at two temperatures.</em> J Clin Pharm Ther, 1993. PMID 8473364.</li>
-<li>Nahata MC et al. <em>Stability of aminophylline in bacteriostatic water for injection stored in plastic syringes at two temperatures.</em> Am J Hosp Pharm, 1993. PMID 1481803.</li>
-<li>Burbridge MA et al. <em>Excipients in Anesthesia Medications.</em> Anesth Analg, 2020. PMID 29505449.</li>
+<li>Nahata MC et al. <em>Stability of aminophylline in bacteriostatic water for injection stored in plastic syringes at two temperatures.</em> Am J Hosp Pharm, 1992. PMID 1481803.</li>
+<li>Burbridge MA et al. <em>Excipients in Anesthesia Medications.</em> Anesth Analg, 2019. PMID 29505449.</li>
 </ol>